src/main/java/core/basesyntax/model/Action.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.model;
+
+public enum Action {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
```

Comment:
It's a bad practice to make your class exposed. Consider using private constructors for enums to prevent the instantiation of the enum class.